agent/05_agent_configure.sh

```diff
@@ -32,6 +32,12 @@ function add_dns_entry {
     if ! $(sudo virsh net-dumpxml ${BAREMETAL_NETWORK_NAME} | xmllint --xpath "//dns/host[@ip = '${ip}']" - &> /dev/null); then
       sudo virsh net-update ${BAREMETAL_NETWORK_NAME} add dns-host  "<host ip='${ip}'> <hostname>${hostname}</hostname> </host>"  --live --config
     fi
+
+    if [[ $NUM_MASTERS == 1 ]]; then
+      echo "${ip} console-openshift-console.apps.${CLUSTER_DOMAIN}" | sudo tee -a /etc/hosts
```

Comment:
I would suggest removing these three entries when "make clean" is called.

Reply:
Done